internal/client.go

```diff
@@ -350,6 +350,10 @@ type (
 	}
 
 	// RetryPolicy defines the retry policy
+	// Note that the history of activity with retry policy will be different: the started event will be written down into
```

Comment:
Add a blank (comment) line or put a period on the end of the first sentence. Godoc will mash the lines together in one paragraph.

Reply:
Period added. Blank line will break the lint check as it requires comment starting with "ReplyPolicy"